visibility/test/test_sun_position/SunPositionAndHillshadeUnitTest.py

```diff
@@ -0,0 +1,127 @@
+
+import unittest
+import logging
+import arcpy
+from arcpy.sa import *
+import sys
+import traceback
+import datetime
+import os
+import TemplateConfigTest
+import TestUtilities
+                
+                
+class SunPositionAndHillshadeUnitTest(unittest.TestCase):
+    def setUp(self):
```

Comment:
Anything you want to do just once per class - put in setUpClass() method (you might only want to do this config check once - although I know we discussed changing this anyway)